test/unit/test_lambda.py

```diff
@@ -0,0 +1,293 @@
+import pytest
+import json
+
+# make sure we can find the app code
+import sys, os
+my_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, my_path + '/../../code/')
+
+import lambda_handlers as handlers
+from jsonschema.exceptions import ValidationError
+
+
+def test_fv_handler():
+    response = handlers.fv_handler(json.load(open('../fv.json')), None)
+    assert 'result' in response
+    assert round(response.get('result'), 6) == 15692.928894
+
+    # TODO test data types
+    # Missing rate
```

Comment:
nit: Better way to document this would be to have separate unit tests for each scenario with the test function named appropriately for the scenario. Then they're self-documenting and show up in the py.test output. I probably set a bad example with my tests, but one assert per test method is considered best practice. I like to exercise a little judgement and combine like I did, however for these different error cases, I think separate test functions would be better.

Reply:
I agree. I'll split them out.